gensim/models/ensemblelda.py

```diff
@@ -0,0 +1,1304 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Authors: Tobias Brigl <github.com/sezanzeb>, Alex Salles <alex.salles@gmail.com>,
+# Alex Loosley <aloosley@alumni.brown.edu>, Data Reply Munich
+# Copyright (C) 2021 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""Ensemble Latent Dirichlet Allocation (eLDA), a method of training a topic model ensemble.
```

Comment:
Same comment – more motivation please, more "why should I care?".

In terms of style, please use imperative mode and full sentences. E.g. "Extract reliable topics that …"

Reply:
I added "for higher reproducibility and less noise" earlier, you mean I should explain a bit why that is a good thing?

Reply:
Definitely. "Higher reproducibility and less noise" is way too abstract. Readers won't know when or why to use this module, what problem it solves (or doesn't solve) for them.

Reply:
I think the new formulation @sezanzeb and I wrote here offers an improvement.  What do you think?